src/components/PermalinkField/PermalinkField.md.scss

```diff
@@ -0,0 +1,27 @@
+.tm-permalink-field-example {
+  .tm-permalink-field {
+    border: 1px solid $gray;
+    position: relative;
+    margin: auto;
+    width: 400px;
+    height: 30px;
+
+    input {
+      padding: 0 0 0 15px;
+    }
+
+    div[role='button'] {
+      border-left: 1px solid $gray;
+      width: 30px;
+      display: flex;
+      align-items: center;
+      justify-content: center;
```

Comment:
this button's css should go in PermalinkField.scss.   Have a look to SearchInput default css. The css are pretty similar.